src/client/src/ui/blotter/model/blotterModel.js

```diff
@@ -69,6 +69,9 @@ export default class BlotterModel extends ModelBase {
               else {
                 this.trades.unshift(trade);
               }
+              if (!trade.isStateOfTheWorld){
+                this._blotterService.createNotification(trade);
```

Comment:
we should have a separate notification service OR just directly call openFin here.

I initially thought we showed in app notifications, but since we don't perhaps just directly calling openFin here makes sense for now.